app/controllers/EventSummaryController.scala

```diff
@@ -37,15 +37,17 @@ import scala.concurrent.{ExecutionContext, Future}
 class EventSummaryController @Inject()(
                                         val controllerComponents: MessagesControllerComponents,
                                         identify: IdentifierAction,
+                                        getData: DataRetrievalAction,
+                                        requireData: DataRequiredAction,
                                         connector: EventReportingConnector,
                                         formProvider: EventSummaryFormProvider,
                                         view: EventSummaryView
                                       )(implicit ec: ExecutionContext) extends FrontendBaseController with I18nSupport {
 
   private val form = formProvider()
 
-  private def summaryListRows(waypoints: Waypoints)(implicit request: IdentifierRequest[AnyContent]): Future[Seq[SummaryListRow]] = {
-    connector.getEventReportSummary(request.pstr).map { seqOfEventTypes =>
+  private def summaryListRows(waypoints: Waypoints)(implicit request: DataRequest[AnyContent]): Future[Seq[SummaryListRow]] = {
+    connector.getEventReportSummary(request.pstr, "21/01/22").map { seqOfEventTypes =>
```

Comment:
What is the significance of the date here?

Reply:
Fixed